### ISSUE TITLE
Solution: Request Parameter Recipient is Missing & keyword `json` not supported

### ISSUE THREAD

**Issue by Chen-Han** [masnun/fb-bot#2]
Hi there, great post, I really enjoyed reading it. 
I had a small problem replying messages.
The line #14 
`requests.post("https://graph.facebook.com/v2.6/me/messages?access_token=" + ACCESS_TOKEN, json=data)`
does not work in my case, it seems like python `requests` module that I am using does not have keyword arg `json`, also, the request made must contain a `Content-Type:Application/json` header
other wise, an error reminding request parameter `recipient` is missing is returned from facebook
this one works

```
import json
resp = requests.post("https://graph.facebook.com/v2.6/me/messages?access_token=" + ACCESS_TOKEN, data=json.dumps(data), headers={'Content-Type':'Application/json'})
```

Hope it helps anyone else reading it.

**Comment by masnun** [comment]
Can you not upgrade the version of `requests` module?

**Comment by Chen-Han** [comment]
Yes I can, but I think it's good to make a note here when someone else is searching for a solution. 
I changed the title a bit so it's easier to search.
